scripts/zones/Bearclaw_Pinnacle/IDs.lua

```diff
@@ -17,9 +17,13 @@ zones[xi.zone.BEARCLAW_PINNACLE] =
         CARRIED_OVER_POINTS       = 7001, -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY   = 7002, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!<space>
         LOGIN_NUMBER              = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
-        PARTY_MEMBERS_HAVE_FALLEN = 7403, -- All party members have fallen in battle. Now leaving the battlefield.
+        PARTY_MEMBERS_HAVE_FALLEN = 7406, -- All party members have fallen in battle. Now leaving the battlefield.
```

Comment:
Need to update these IDs for current client.  This is actually wrong previously, and as of May is 7407.  The other additions are also incorrect (Blown away is 7615, for example)

Reply:
I was on the most up to date client when I checked this o.o. the texts came out just fine.

Reply:
But i'll double check